### PR TITLE
Hivemind rebalance fuckery part 2 the reckoning.

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

Hivemind balance is once more thrown off the rails.

This PR includes name and description changes, primarly capitalizing names
Along with buffs and nerfs for the hivemind, such as:

Reducing maneater (now Mechiver) HP and (a bit) damage dealing via its special ability
Removed certain useless comments I made before
Nerfs tox damage of the goo projectile
Reduces cooldown of tormentor (head on a tendril) while also greatly increasing cooldown of shockwave ability for the core
Tweaking rarity_value of mobs
Wires now deal more damage to doors and have a higher chance of unlocking them
Increases time for hivemind to be announced
Fixed ressurected mobs description

This PR also prepares room for a new hivemob in the future, along with a projectile it will be firing
At the moment the code does nothing, the comment in the mobs file is more of a to do/reminder

## Why It's Good For The Game

Certain parts of hivemind were unbalanced, both for the benefit of the crew and harm.
This PR changes that, in the form of nerfing certain parts of the event (such as Mechivers, Hive core shockwave) and buffing others (increasing damage dealt to doors by wires, increasing time until hive is announced)
There are also non-gameplay changes, such as the names, descriptions, and comments in the code

## Changelog
:cl:
add: New attack text for Phasers
tweak: Hivemobs names are now same as their code names, and changed some of their descriptions. Hive is also now announced later. Rarity value of mobs is now also changed
balance: Hivemind shockwave cooldown is greater, tormentor cooldown is shorter. Goo also deals less tox damage now.  Assemblers are no longer dense, mobs can move through them. Mechiver has less HP and deals a bit less damage via its special ability. Wires now deal more damage to doors and have a higher chance of unbolting them
sounddel: removed outbreak7.ogg from hivemind announcement
code: Removed/edited some comments I left
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
hivemind will free us all

File: code/modules/hivemind/objects.dm [hunk 1]
```diff
@@ -3,20 +3,24 @@
...
+	name = "Weakened Electrolyzed goo"
+	damage_types = list(BURN = 5)
+
 
 /obj/item/projectile/goo/on_hit(atom/target)
 	. = ..()
 	if( isliving(target) && !issilicon(target) )
 		var/mob/living/L = target
-		L.damage_through_armor(15, TOX, attack_flag = ARMOR_RAD)
+		L.damage_through_armor(5, TOX, attack_flag = ARMOR_RAD)
```
Comment (TheShown911) [inline code/modules/hivemind/objects.dm:22]: With this damage and something like a technomancer voidsuit the projectiles would do no damage with GDR.

Comment (Wouju) [reply code/modules/hivemind/objects.dm:22]: should I increase it (either back to 15 or a different value) then?

Comment (TheShown911) [reply code/modules/hivemind/objects.dm:22]: I'd bump it up to 10 and change the armor type to energy.

Comment (Wouju) [reply code/modules/hivemind/objects.dm]: Done so

**Review by mattroks101 — APPROVED** [review]

**Pull request merged**